src/client/pages/ApplyPage/components/PersonalSection.tsx

```diff
@@ -226,42 +235,51 @@ class PersonalSection extends ApplyPageSection<PersonalSectionFormData, Personal
   renderInstitutionOptions(allowHighSchool: boolean) {
     return (
       <span>
-        {FormFields.createRow(
-          FormFields.createColumn('col-sm-12 no-margin-bottom',
-            FormFields.createLabel('Institution')
-          ),
-          FormFields.createColumn('col-md',
-            this.createInstitutionCard(InstitutionType.UCSD, 'institution-ucsd',
-              'UCSD')
-          ),
-          FormFields.createColumn('col-md',
-            this.createInstitutionCard(InstitutionType.University, 'institution-uni',
-              'Other University')
-          ),
-          allowHighSchool ? FormFields.createColumn('col-md',
-            this.createInstitutionCard(InstitutionType.HighSchool,
-              'institution-hs', 'High School')
-          ) : '',
-          FormFields.createColumn('col-sm-12',
+        <ApplicationRow>
+          <ApplicationCol className="col-sm-12 no-margin-bottom">
+            <ApplicationLabel>Institution</ApplicationLabel>
+          </ApplicationCol>
+
+          <ApplicationCol className="col-md">
+            {this.createInstitutionCard(InstitutionType.UCSD, 'institution-ucsd',
```

Comment:
These can probably be migrated too, right?

Reply:
I thought about it, but I think the real intent behind this refactor is to remove the nested function calls to make things more composable, didn't see a real use for "display components" to be migrated away as yet

Reply:
I agree that's the major selling point of this refactor, but having a `createXYZ` function that generates a component is basically just a constructor of a stateless component with extra steps and less readability.